### DISCUSSION TITLE
Release 0.9.70 discussion

### DISCUSSION THREAD

**Discussion by kmk3** [netblue30/firejail#5181] [General]
Release links:

* Project board: https://github.com/users/netblue30/projects/3
* GitHub Release: https://github.com/netblue30/firejail/releases/tag/0.9.70

Discussion for the previous release:

* #4770

Discussion for the next release:

* #5192

---

From commit e8cb03cde ("I am preparing a point release for next week, fixes and
small number of new features. Check in everything you have out.", 2022-06-03)
by netblue30:

> I am preparing a point release for next week

When exactly is the intended release date?

I'd like to fix an issue caused by #5142 and to finish the RELNOTES beforehand.

Relates to #5174.

**Comment by netblue30** [comment]
Sure, put all the fixes in - release sometime this week!

**Comment by rusty-snake** [comment]
Tracking this J4F

- [x] [Debian bullseye](https://packages.debian.org/bullseye/firejail) (CVE-2022-31214 backported)
- [ ] [Ubuntu 20.04 LTS jammy](https://packages.ubuntu.com/jammy/firejail)
- [ ] [Fedora 36](https://packages.fedoraproject.org/pkgs/firejail/firejail/fedora-36.html)
- [ ] [OpenSUSE Leap 15.3](https://build.opensuse.org/package/show/openSUSE%3ALeap%3A15.3/firejail)
- [x] [OpenSUSE Tumbleweed](https://build.opensuse.org/package/show/openSUSE%3AFactory/firejail) (CVE-2022-31214 backported 2022-06-09)
- [x] [ArchLinux](https://archlinux.org/packages/community/x86_64/firejail/)
- [x] [Manjaro](https://software.manjaro.org/package/firejail)

2022-06-09: Firejail 0.9.70 released
2022-06-11: Create post and add ArchLinux
2022-06-15: Add Manjaro
2022-06-18: Add OpenSUSE Tumbleweed CVE-2022-31214 backport
2022-07-09: Debian has backported CVE-2022-31214 in the mean time

**Comment by EdiDD** [comment]


**Reply by rusty-snake** [reply]
- System-wide: You can write to `/proc/sys/kernel/seccomp/actions_logged` but you likely don't want.
- System-wide: Maybe there are some other configuration options?
- Per-profile: You can allow the blocked syscall-calls but cosmetic reasons are bad reasons for weaking the sandbox.
- Revert+Compile
- Good way: none

What do you (all) think about something like

```C
int seccomp_load(struct sock_fprog *prog) {
#ifdef SECCOMP_FILTER_FLAG_LOG
    if (arg_debug)
        return syscall(SYS_seccomp, SECCOMP_SET_MODE_FILTER, SECCOMP_FILTER_FLAG_LOG, prog)
#endif
    return prctl(PR_SET_SECCOMP, SECCOMP_MODE_FILTER, prog)
}
```

**Reply by kmk3** [reply]
@EdiDD [on Jun 14](https://github.com/netblue30/firejail/discussions/5181#discussioncomment-2947406):

> There are many log entries like: `audit: SECCOMP ...` and `kernel: audit:
> ...` in journal probably because of: `* feature: always log seccomp errors
> (#5110)` Is there a way to disable this or make these messages silently
> ignored ?

This discussion is about organizing the release in general; to report a
bug or request a feature, please use
<https://github.com/netblue30/firejail/issues/new/choose>.

Creating a specific issue doesn't add noise and ensures that the matter in
question is properly tracked.
